README.md

```diff
@@ -90,6 +90,7 @@ Supported Cars
 | Lexus     | ES Hybrid 2017-18             | LSS               | Stock<sup>3</sup>| 0mph               | 0mph              |
 | Lexus     | ES Hybrid 2019-21             | All               | openpilot        | 0mph               | 0mph              |
 | Lexus     | IS 2017-2019                  | All               | Stock            | 22mph              | 0mph              |
+| Lexus     | IS Hybrid 2017-2019           | All               | Stock            | 22mph              | 0mph              |
```

Comment:
are all these years confirmed working?

Reply:
No, I'll move it back to 2017 for now or whatever @bisra2's ISH is.

Reply:
@bisra2 says his is the 2018 so I will edit the README to just 2018.